docs/playbooks/lite-assist-roks.md

```diff
@@ -0,0 +1,147 @@
+# MAS Core Service on IBM Cloud ROKS
+
+This master playbook will drive the following actions:
+
+- [Provision & setup OCP on IBM Cloud](ocp.md#provision) (20-30 minutes)
+- Install dependencies:
+    - [Install MongoDb (Community Edition)](dependencies.md#install-mongodb-ce) (10 minutes)
+    - [Install SLS](dependencies.md#install-sls) (10 minutes)
+    - [Install UDS](dependencies.md#install-uds) (35 minutes)
+- Install & configure MAS:
+    - Generate MAS Workspace Configuration (1 minute)
+    - [Configure Cloud Internet Services integration](mas.md#cloud-internet-services-integration) (Optional, 1 minute)
+    - [Install & configure MAS](mas.md#install-mas) (15 minutes)
+- [Configure and prepare ibm cloud object storage instance](../roles/cos_setup.md)
+- Install and prepare discovery instance
+    - [Install discovery Service](cp4d.md#install-services-discovery)
+    - [Create Disocvery instance](cp4d.md#create-discovery-instance)
+- Install Assist and Configure Assist Workspace
+    - [Refer to Suite App Assist](../roles/suite_app_install.md) (20 Minutes)
+    - [Refer to Suite App Configure](../roles/suite_app_configure.md)  (10 Minutes)
+
+All timings are estimates, see the individual pages for each of these playbooks for more information.
+
+## Preparation
+Before you run the playbook you **must** prepare the entitlement license key file that will be used during the playbook run.
+
+Copy the MAS license key file that you obtained from Rational License Key Server to `$MAS_CONFIG_DIR/entitlement.lic` (the file must have this exact name).  During the installation of SLS this license file will be automatically bootstrapped into the system.
```

Comment:
update doc on the SLS_LICENSE_FILE var and the prepartion part. See https://github.com/ibm-mas/ansible-devops/blob/master/docs/playbooks/lite-manage-roks.md for example

Reply:
updated.